README.md

```diff
@@ -36,17 +32,14 @@ We focus to solve the web development experience for backend developers and newb
 + Web developers need better performance in their web APPs
   + They can use some sepcified component from our libraray
 
-
 ## Personal connection
-
 > What is your personal connection to the service & the user group you are planning to reach with your product? *
 
+We currently have little idea about this.
 
-## Work on
-> If we do not fund this project, will your team still work on it? To what extent? *
-
-We will work on it also, but it will cost more times because we really need some outside supports now, to prove this project is meaningful and to encourage us working harder, for the next chapter of the internet.
-
++ Develop websites with our library for public service for free.
```

Comment:
This item might be useless. Open source means free to use.

Reply:
One of my friends maybe can help us(like contacting some students in CAFA), we can develop a Gallery for artworks for college and popularize the lib for newbies.

I was trying to explain this...